content/react/pt/get-started.md

```diff
@@ -6,20 +6,20 @@ description: "Configuração do React Storybook no ambiente de desenvolvimento"
 
 # Introdução
 
-Storybook funciona em paralelo á aplicação em modo de desenvolvimento.
-Ajuda na construção de componentes de interface de utilizador isolados de qualquer lógica e contexto da aplicação.
-Esta edição de Aprendizagem de Storybook é destinada para React.
-Encontram-se disponíveis outras edições quer para [Vue](/vue/pt/get-started), quer para [Angular](/angular/pt/get-started).
+Storybook funciona em paralelo à aplicação em modo de desenvolvimento.
+Ajuda na construção de componentes de interface de usuário, isolados de qualquer lógica e contexto da aplicação.
```

Comment:
In this case "Utilizador" or "usuário" is the same same thing. Both portuguese, either portuguese or brazilian portuguese speaking people do understand the meaning of the word.

Reply:
Sorry, it is not common in Brazil to use/read "Utilizador".

Reply:
The word is not common, but it does exist and is recognized for both audiences.